packages/beacon-node/test/unit/network/peers/datastore.test.ts

```diff
@@ -26,7 +26,7 @@ describe("Eth2PeerDataStore", () => {
     expect(dbDatastoreStub.batch.calledOnce).to.equal(true);
   });
 
-  it("should persist to db the oldest item after max", async () => {
+  it.only("should persist to db the oldest item after max", async () => {
```

Comment:
Why is this test skipped? Add a comment above explaining the error

Reply:
needs to be removed